src/modules/main/controllers/admin-read.controller.ts

```diff
@@ -247,6 +247,14 @@ export class AdminReadController {
     return mapper.createNftDetailDto(item);
   }
 
+  @Get('nft/trial/:id')
+  async getWalletTrialDetail(@Param('id', ParseUUIDPipe) id: NftId) {

```

Comment:
we already have endpoints to return nfts, moreover this endpoint is by no means bound to current user

Reply:
I need it not to be for trial mint.